src/selectors/fridge.js

```diff
@@ -234,3 +234,27 @@ export const selectAverageTemperature = createSelector(
     return Number(averageTemperature).toFixed(1);
   }
 );
+
+export const selectSelectedFridgeIsInHotBreach = state => {
+  const fridge = selectSelectedFridge(state);
+  const { isInHotBreach } = fridge;
+  return isInHotBreach;
+};
+
+export const selectSelectedFridgeIsInColdBreach = state => {
+  const fridge = selectSelectedFridge(state);
+  const { isInColdBreach } = fridge;
+  return isInColdBreach;
+};
+
+export const selectSelectedFridgeIsLowBattery = state => {
```

Comment:
(joke) battery powered fridge 🤪 . TBF there are probably battery powered fridges out there but that's not my point.

Reply:
9f1a529

not sure if thats better 👀

Reply:
whoops! 
0a51670 also

Reply:
Was a joke but thanks haha